platform_versioned_docs/version-24.1.1/enterprise/kubernetes.mdx

```diff
@@ -14,6 +14,8 @@ Complete the following procedures to install Seqera Platform Enterprise on a Kub
 
 ### Create a namespace
 
+Seqera uses a dedicated namespace to isolate Kubernetes pods and services from the other resources on your cluster.
```

Comment:
Should we include instructions for removing/replacing the namespace if we go ahead with it? Or should we include instructions for users to add their own?

Reply:
Think this is need. It's not a K8s tutorial. A link to K8s docs can be enough